<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeConditionUI</class>
 <widget class="QDialog" name="ChangeConditionUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Condition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Title">
     <property name="text">
      <string>Change Condition For</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="UpverticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QComboBox" name="ConditionCombo"/>
   </item>
   <item>
    <spacer name="LowverticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="ButtonsLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>OkButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeConditionUI</receiver>
   <slot>modifyCondition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeConditionUI</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modifyCondition()</slot>
 </slots>
</ui>
